cleanlab_studio/studio/studio.py

```diff
@@ -118,12 +118,13 @@ def apply_corrections(self, cleanset_id: str, dataset: Any) -> Any:
             else:
                 joined_ds = dataset.join(cl_cols.set_index(id_col).sort_values(by=id_col))
             joined_ds["__cleanlab_final_label"] = joined_ds["clean_label"].where(
-                joined_ds["clean_label"] != "None", dataset[label_column]
+                np.asarray(list(map(check_not_none, joined_ds["clean_label"].to_numpy()))),
```

Comment:
nit: I believe you can just do `np.asarray(list(map(check_not_none, joined_ds["clean_label"])))`